### ISSUE TITLE
Updates and pulls

### ISSUE THREAD

**Issue by DavidGriffith** [Quuxplusone/Castlequest#4]
It seems that when you're doing updates, you're doing it in such a way that typing `git pull` doesn't cleanly work.  Instead I'm asked to perform a merge.  Are you doing something like `git push -f origin`?

**Comment by Quuxplusone** [comment]
Yes. I'm a rebase fiend. The benefit is that the history of `main` will always be clean; people who arrive "late" to the party won't have to deal with a whole bunch of little commits.

**Comment by DavidGriffith** [comment]
The way I avoid that is to always do my whole bunch of little commits in a branch.  Then I merge, squash down, or rebase the commits, and only then do a push.

**Comment by Quuxplusone** [comment]
The good news is that once we have ironed out all the (non-original) typos in my transcription, such that the transcription is perfect, there'll be no more rewriting. So there *is* at least a well-defined end state we're trying to get to. :)
